app/routes/core.rb

```diff
@@ -21,7 +21,7 @@ class Core < Sinatra::Application
         enable :show_exceptions
       end
 
-      unless settings.test?
+      unless settings.test? || settings.development?
```

Comment:
A post with reproduction instructions and what your expectations are would be great 👍